GenderPayGap.Database/GpgDatabaseContext.cs

```diff
@@ -16,7 +16,8 @@ namespace GenderPayGap.Database
     public partial class GpgDatabaseContext : IDbContext
     {
 
-        public static string ConnectionString = @"Server=(localdb)\ProjectsV13;Database=GpgDatabase;Trusted_Connection=True;";
+//        public static string ConnectionString = @"Server=(localdb)\ProjectsV13;Database=GpgDatabase;Trusted_Connection=True;";
+        public static string ConnectionString = @"Server=127.0.0.1;Port=5432;Database=gpg_database;User Id=gpg_user;Password=local_gpg_database;";

```

Comment:
I'd like to be able to switch between using SQL Server and Postgres locally easily.
e.g.
* when you're working on PaaS/DB stuff, you can be using Postgres
* when I'm doing stuff that's going to be deployed to Azure, I can still use SQL Server.

Would it help to make this switch between the two connections strings based on a setting?
Or is that handled somewhere else?

Reply:
This is now handled in appsettings.local.json by two values, `UsePostgresDb` and `  "ConnectionStrings": {
    "GpgDatabase": "Server=127.0.0.1;Port=5432;Database=GpgDatabase;User Id=gpg_user;Password=local_gpg_database;"
  }`

Reply:
It is set to false by default in appsettings.json, then overwritten in the local file